tests/test_downloader_handlers.py

```diff
@@ -287,6 +288,11 @@ def test_redirect_status_head(self):
 
     @defer.inlineCallbacks
     def test_timeout_download_from_spider_nodata_rcvd(self):
+        if self.reactor_pytest == "asyncio" and sys.platform == "win32":
+            raise unittest.SkipTest(
+                "This test produces DirtyReactorAggregateError on Windows with asyncio"
```

Comment:
What about including a link to the upstream issue you reported when skipping these tests, for future reference? (either as a code comment or in the skip message).